editor/action_map_editor.cpp

```diff
@@ -56,16 +56,34 @@ void ActionMapEditor::_event_config_confirmed() {
 	Dictionary new_action = current_action.duplicate();
 	Array events = new_action["events"].duplicate();
 
+	// Check for duplicate events on action
+	for (Ref<InputEvent> event : events) {
+		if (ev->is_match(event, true)) {
+			if (accept_warning->is_connected(SceneStringName(confirmed), callable_mp(this, &ActionMapEditor::_event_add_edit_confirmed))) {
+				accept_warning->disconnect(SceneStringName(confirmed), callable_mp(this, &ActionMapEditor::_event_add_edit_confirmed));
+			}
+
+			accept_warning->set_text(TTR(vformat("Action \"%s\" already has an Event with this Input.", current_action_name)));
```

Comment:
The order of TTR and vformat needs to be changed. TTR translates the text, if the input action is inserted before that it won't find the corresponding translation.